core/src/main/java/nl/nn/adapterframework/validation/xsd/ResourceXsd.java

```diff
@@ -43,7 +43,7 @@ public void initNoNamespace(IScopeProvider scopeProvider, String resourceRef) th
 	public void initNamespace(String namespace, IScopeProvider scopeProvider, String resourceRef) throws ConfigurationException {
 		this.resourceRef = resourceRef.replace("%20", " ");
 
-		resource = Resource.getResource(scopeProvider, resourceRef);
+		resource = Resource.getResource(scopeProvider, this.resourceRef);
```

Comment:
And this is why it's so important to create tests...

Reply:
The code was written in an unclean way, because the var names should not overlap.  Causing such hard to find issues. But tests are useful, yes.